packages/apollo-server/README.md

```diff
@@ -122,17 +122,17 @@ new ApolloServer({
 
 ## Documentation
 
-The [Apollo Server documentation](https://apollographql.com/docs/apollo-server/) contains many of the details which are necessary to get started with both GraphQL and Apollo Server.
+The [Apollo Server documentation](https://apollographql.com/docs/apollo-server/) contains much more to get started with GraphQL and Apollo Server.
 
-The raw source content of that documentation is available within the `docs/` directory of this monorepo, but the _Edit on GitHub_ buttons at the bottom of each page can be used to contribute suggestions or improvements to the published content.
+The raw source of said documentation is available within the `docs/` directory of this monorepo--to contribute, please use the _Edit on GitHub_ buttons at the bottom of each page.
 
 ## Development
 
-If you want to develop or contribute to Apollo Server itself, we suggest the following:
+If you wish to develop or contribute to Apollo Server, we suggest the following:
 
 - Fork this repository
 
-- Install the Apollo Server project in your computer
+- Install Apollo Server
```

Comment:
This isn't quite right.  Installing Apollo Server isn't the path to _developing_ it, it's the path to _using_ it.  This should read more like, "Clone this repository into your development environment." (whatever that might be).